doc/content/reference/gs-mqtt-protocol/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Gateway Server MQTT Protocol"
+description: "Gateway Server MQTT Protocol"
```

Comment:
Browsing in the local doc server, I'd find this more appropriate under API, and that it shows up as "Gateway Server MQTT"

Reply:
Given that this not a user-facing service, does it make sense for it to appear next to the gRPC and HTTP APIs?

Reply:
Define end-users. The target audience for the API section is developers, and those can be of all sorts: packet forwarder developers and app developers to manage applications in TTES.

Reply:
Alright